libraries/chain/controller.cpp

```diff
@@ -1820,6 +1820,8 @@ struct controller_impl {
                   ("b", blog.first_block_num())("e", blog_head->block_num()) );
             block_states = read_from_snapshot( snapshot, blog.first_block_num(), blog_head->block_num() );
          } else {
+            EOS_ASSERT( !fork_db.file_exists(), snapshot_exception,
```

Comment:
Nothing wrong with the snapshot. I think this should be a `fork_database_exception`.

Reply:
Thanks, done.